server/views/pages/bookedtoday/arrivals/searchForExistingRecord/searchForExistingRecord.njk

```diff
@@ -26,31 +24,26 @@
                         actions: { items: [
                             { href: baseUrl + '/change-date-of-birth', text: "Change", visuallyHiddenText: "date of birth" }]}
                     }
-                ]%}
-
+                ] %}
                 {{ govukSummaryList({ rows: details }) }}
-                
                 <h3 class="govuk-heading-m">Identifiers</h3>
-
                 {% set identifiers = [
                     {   key:     { text: "PNC number" },
-                        value:   { text: data.pncNumber | default('Not entered') },
+                        value:   { text: data.pncNumber | default('Not entered', true) },
```

Comment:
ensure that if the user saves an empty value the default message shows. By default only shows message if strictly undefined